Framework/DataHandling/src/LoadEventAsWorkspace2D.cpp

```diff
@@ -90,6 +90,9 @@ void LoadEventAsWorkspace2D::init() {
                   "The name of the units to convert to (must be one of those registered in the Unit Factory)");
   declareProperty(std::make_unique<WorkspaceProperty<Workspace2D>>("OutputWorkspace", "", Direction::Output),
                   "An output workspace.");
+  declareProperty("LoadNexusInstrumentXML", true,
+                  "If true, load the instrument XML file. If the instrument XML file is not needed, setting this to "
```

Comment:
That is not what this option does. If set to false it will skip trying to load the instrument from the NeXus file first and just try to load the local copy of the instrument XML. So the instrument is loaded regardless if `true` or `false`, just where it is loaded from changes.

Reply:
Updated